docs/_partner_api/endpoints.md

```diff
@@ -347,6 +350,38 @@ curl -H "Authorization: Bearer {token}" -L "https://api.zapier.com/v1/zaps?&para
 }
 ```
 
+## GET /v1/profiles/me
+
+**Example Requests**
+
+Get user information related to the given `access_token`
+
+```bash
+curl -H "Authorization: Bearer {token}" -L "https://api.zapier.com/v1/profiles/me"
+```
+
+**Example Response**
+
+> Refer to the [Profile](#profile) reference for all the available fields.
+
+```json
+{
+    "id": 88998899,
+    "first_name": "Jacob",
+    "last_name": "Corwin",
+    "email": "jacob.corwin@gmail.com",
+    "email_confirmed": true,
+    "timezone": null,
+    "full_name": "Jacob Corwin",
+    "roles": [
```

Comment:
Now that we've decided to not support the `account_id` query parameter in `v1/zaps` I'm thinking we should remove the `roles` attribute from the response payload of `profiles/me`.  What are your thoughts?  It would be 1-3 lines of code to easily change.

Reply:
Sounds good! I'll remove it on https://github.com/zapier/partnerapi/pull/102

Reply:
The roles attribute gets added in the monolith.  I can make that change and toss it up as a PR for you (maybe tomorrow).